app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -1,343 +1,320 @@
-  ```
-  $ sudo vi /var/lib/pgsql/data/pg_hba.conf
-  ```
+2. Initialize the PostgreSQL database and enable automatic start.
 
-2. Change the `IPV4` local connection method to `md5`
-  ```
-    | TYPE | DATABASE | USER | ADDRESS      | METHOD |
-    | # IPv4 local connections:                      |
-    | host | all      | all  | 127.0.0.1/32 |  md5   |
-    | # IPv6 local connections:                      |
-    | host | all      | all  | ::1/128      | ident  |
-  ```
+    ```bash
+    $ sudo /usr/bin/postgresql-setup --initdb
+    $ sudo systemctl enable postgresql.service
+    $ sudo systemctl start postgresql
+    ```
+    
+3. Switch to PostgreSQL user and launch PostgreSQL.
 
-3. Save and exit the file, then restart Postgres
-  ```
-  $ sudo systemctl restart postgresql
-  ```
+    ```bash
+    $ sudo -i -u postgres
+    $ psql
+    ```
 
-Now that the database is set up, Kong must be given credentials to access the database. This is done by adding the Postgres user and password information to Kong’s configuration file.
+4. Create a Kong database with a username and password.
 
+  > ⚠️**Note**: Make sure the username and password for the Kong Database are
+  > kept safe. We have used a simple username and password for illustration purposes only. Note the database name, username and password for later. 
 
-1. Make a copy of Kong’s configuration file
-  ```
-  sudo cp /etc/kong/kong.conf.default kong.conf
-  ```
+    ```bash
+    $ psql> CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
+    ```
 
-2. Open the new configuration file
-  ```
-  sudo vi /etc/kong/kong.conf
-  ```
+5. Exit from PostgreSQL and return to your terminal account.
 
-3. Search for `pg_password` under `DATASTORE,` uncomment `pg_password` and add the value. 
-  ```
-  #pg_user = kong
-  pg_password = kong
-  #pg_datastore = kong
-  ```
-  > Note: If you used different values for the user and database name, uncomment the pg_user and pg_datastore variables and add the username and datastore values
+    ```bash
+    $ psql> \q
+    $ exit
+    ```
 
+6. Edit the the PostgreSQL configuration file `/var/lib/pgsql/data/pg_hba.conf` using your preferred editor.
 
-## Step 4. Customize Your Configuration
-Before starting Kong, you can further modify Kong’s configuration file to enable a host of different features:
+    Under IPv4 local connections replace `ident` with `md5`:
 
+    | Protocol   	| Type 	| Database 	| User 	| Address      	| Method 	|
+    |------------	|------	|----------	|------	|--------------	|--------	|
+    | IPv4 local 	| host 	| all      	| all  	| 127.0.0.1/32 	| md5    	|
+    | IPv6 local 	| host 	| all      	| all  	| 1/128        	| ident  	|
 
-### Enable RBAC
+    PostgreSQL uses `ident` authentication by default. To allow the `kong` user to communicate with the database locally, we must change the authentication method to `md5` by modifying the PostgreSQL configuration file. 
```

Comment:
Remove "we must". Change:
"PostgreSQL uses `ident` authentication by default. To allow the `kong` user to communicate with the database locally, we must change the authentication method to `md5` by modifying the PostgreSQL configuration file." 
to
"PostgreSQL uses `ident` authentication by default. To allow the `kong` user to communicate with the database locally, change the authentication method to `md5` by modifying the PostgreSQL configuration file. "